middleware/middleware-1.3.35.md

```diff
@@ -0,0 +1,49 @@
+---
+slug: /release-notes/middleware/1.3.35
+title: Version 1.3.35
+---
+
+# fiskaltrust.Middleware 1.3.35 (Germany)
+_July 05, 2022_
+
+In this version of the Middleware, we removed non needed properties for FCC model classes, we also solved several bugs and created a FCC container for our _Bring your own Data Center_ solution.
+
+:::caution
+
+Version 1.3 of the Middleware is meant for the German market only, customers in Austria and France should continue to use version 1.2. We will unify these experiences in an upcoming version.
+
+:::
+
+## Feature: Removal of cspSystemTime property for Swissbit Cloud & Deutsche Fiskal
+We modified the cspSystemTime property for  _SwissbitCloud_ and _DeutscheFiskal_ TSEs to optional. In some rare cases having this property led to exceptions. Additionnally we removed all non-needed properties for FCC model classes.
+
+## Feature:  DSFinV-K export fallback in case of empty "ITemCaseName"
+If the field ZAHLART is empty because of an empty ITemCaseName in ftReceiptCaseData, we automated that it would be filled with the payitem descrption. This will enable auditors to distinguish between the different payment types.
```

Comment:
```suggestion
If the field ZAHLART is empty because of an empty `ItemCaseName` property in `ftReceiptCaseData`, we automated that it would be filled with the PayItem's description. This will enable auditors to distinguish between the different payment types.
```